api/host.py

```diff
@@ -243,6 +243,7 @@ def delete_hosts_by_filter(
 
 
 def _delete_host_list(host_id_list, rbac_filter):
+    request_host = flask.request.headers.get("Host", "")
```

Comment:
This may be renamed to another thing? My understand is that this value would be the consoleDOT URL?
```suggestion
    request_host_url = flask.request.headers.get("Host", "")
```

Reply:
@jpramos123 I think it's just the hostname of the request, so maybe `request_hostname`?

Reply:
@kruai Yup, makes sense